files/galaxy/dynamic_job_rules/production/total_perspective_vortex/tools.yml

```diff
@@ -550,7 +550,10 @@ tools:
   toolshed.g2.bx.psu.edu/repos/iuc/poretools_yield_plot/poretools_yield_plot/.*:
     cores: 3
   toolshed.g2.bx.psu.edu/repos/iuc/qualimap_bamqc/qualimap_bamqc/.*:
-    cores: 5
+    - if: 0.2 <= input_size < 30
```

Comment:
there is one line missing here:
`    rules:`

Reply:
Thank you! I added 'rules:'